lib/utils/errorModel.js

```diff
@@ -188,7 +209,7 @@ function getErrorDetails (cwd, error) {
 const getErrorModel = function (error) {
   flagError(error)
   const viewObject = {
-    errorStack: error.stack,
+    errorStack: unstyle(error.stack),
```

Comment:
If it's only SASS errors that need unstyling (so far) I think we should move this into `sassMatcher()`.

Reply:
I would argue that it doesn't hurt where it is and it covers us for if we ever come across any other errors that are styled.

Reply:
I'd rather keep it more contained - otherwise things like this tend to be left in through fear or what would happen if they're removed.

Reply:
ok

Reply:
I've moved the unstyling of the error stack into the sass matcher as requested but then had to add some extra scafolding to cater for the fact that we don't return the stack from each matcher.  To be honest I like the cleaner version we had before, but I've made the change as you asked.